src/tests/simulation/tests/integration_tests/scale/sink.rs

```diff
@@ -16,10 +16,10 @@ use std::collections::HashMap;
 use std::time::Duration;
 
 use anyhow::Result;
-use madsim::export::futures::StreamExt;
-use madsim::rand::prelude::SliceRandom;
-use madsim::rand::thread_rng;
-use madsim::time;
+use tokio::export::futures::StreamExt;
+use tokio::rand::prelude::SliceRandom;
+use tokio::rand::thread_rng;
```

Comment:
These seem not right?
```suggestion
use futures::StreamExt;
use rand::prelude::SliceRandom;
use rand::thread_rng;
```

Reply:
Nice catch. It's even not included in the module tree. 🥵